src/tests/ftest/osa/online_drain.py

```diff
@@ -49,8 +51,12 @@ def run_online_drain_test(self, num_pool, oclass=None, app_name="ior"):
         targets = int(self.server_managers[-1].get_config_value('targets'))
         t_string = ','.join(map(str, self.random.sample(range(targets), 2)))
 
-        # Drain one of the ranks (or server)
-        rank = self.random.choice(list(self.server_managers[0].ranks.keys()))
+        # For testing with multiple ranks as dmg parameters, use a list of ranks.
+        if multiple_ranks is None:
+            # Drain one of the ranks (or server)
+            rank = self.random.choice(list(self.server_managers[0].ranks.keys()))
+        else:
+            rank = multiple_ranks
```

Comment:
Since single rank uses random, it's better to also use random for multiple ranks. In production, we can't control which rank fails, so it's better to use random. E.g. for 2 random ranks
```suggestion
        # For testing with multiple ranks as dmg parameters, use a list of ranks.
        if multiple_ranks is None:
            # Drain one of the ranks (or server)
            rank = self.random.choice(list(self.server_managers[0].ranks.keys()))
        else:
            rank = ",".join(map(str, random.sample(list(self.server_managers[0].ranks.keys()), k=2)))
```

Reply:
Done.